deploy/docker/docker-compose-noaws.yml

```diff
@@ -0,0 +1,78 @@
+version: '3'
```

Comment:
With the introduction of this compose file, can we just remove the top level docker-compose.yml file?